instrumentation/http/src/main/java/brave/http/HttpClientRequest.java

```diff
@@ -40,11 +40,11 @@
   }
 
   /**
-   * Sets a request header with the indicated name. Null values are unsupported.
+   * Sets a request header with the indicated name. {@code null} values are unsupported.
    *
-   * This is only used when {@link TraceContext.Injector#inject(TraceContext, Object) injecting} a
-   * trace context as internally implemented by {link HttpClientHandler}. Calls during sampling or
-   * parsing are invalid.
+   * <p>This is only used when {@link TraceContext.Injector#inject(TraceContext, Object) injecting}
+   * a trace context as internally implemented by {link HttpClientHandler}. Calls during sampling or
+   * parsing are invalid and may be ignored.
```

Comment:
Maybe can add `may be ignored by the instrumentation.`

I first read this as a user ignoring something and got confused.

Reply:
yeah that's better.